spec/02-integration/02-cmd/02-start_stop_spec.lua

```diff
@@ -57,6 +57,20 @@ describe("kong start/stop #" .. strategy, function()
     assert(helpers.kong_exec("start --conf " .. helpers.test_conf_path))
     assert.truthy(helpers.path.exists(helpers.test_conf.kong_env))
   end)
+
+  if strategy == "cassandra" then
+    it("start resolves cassandra contact points", function()
+      assert(helpers.kong_exec("start --prefix " .. helpers.test_conf.prefix, {
+        database = strategy,
+        cassandra_contact_points = "localhost",
+        cassandra_keyspace = helpers.test_conf.cassandra_keyspace,
+      }))
+      assert(helpers.kong_exec("stop", {
+        prefix = helpers.test_conf.prefix
```

Comment:
It shouldn't be necessary to specify the prefix here (`helpers.test_conf.prefix` already is the default for tests), unless we specify it in both `start` and `stop` (so as to reduce confusion, for explicitness' sake).

Reply:
removed.